ansible/roles/nginx/vars/cchq_ssl.yml

```diff
@@ -20,6 +25,24 @@ nginx_sites:
    add_header: "X-Frame-Options SAMEORIGIN"
    access_log: "{{ log_home }}/{{ deploy_env }}-timing.log timing"
    locations:
+    - name: "^~ /(home|impact|pricing|services|software|solutions|askdemo)$"
```

Comment:
I'm confused what `^~ ` is supposed to mean here. According to http://nginx.org/en/docs/http/ngx_http_core_module.html#location

> If the longest matching prefix location has the “^~” modifier then regular expressions are not checked.

it seems to mean "don't check regular expressions", but this seems like definitely a regular expression. (Maybe I interpreted that sentence wrong—there's not much there.

The only other thing I can think of is that you transposed the `^`, `~`, and ` ` characters and meant

```
"~ ^/(home|impact|pricing|services|software|solutions|askdemo)/$"
```
in which case the tilde means "this is a regular expression, and the `^` is just part of the regular expression.